addons/mail/static/src/models/activity/activity.js

```diff
@@ -204,6 +204,9 @@ function factory(dependencies) {
             this.thread.refresh();
             const thread = this.thread;
             this.delete();
+            if (!action) {
+                return thread.refreshActivities();
```

Comment:
`return` after refresh. This makes it look like the return value of refresh is important, but it's not (I don't think we want to wait for the thread to be refreshed to consider `markAsDoneAndScheduleNext` "finished".

Can you explain a bit more what is the flux where we get false here? Feels wrong to me that an action named `action_feedback_schedule_next` can return something that is not allowing to schedule anything else.

Reply:
@seb-odoo yes there are two cases for `action_feedback_schedule_next` 

1. The action_feedback creates its own next activity (for example when `force_next` is `True`[1]). In that case `action_feedback_schedule_next` will return `False` [2].
2. The action_feedback doesn't aumatically create a new activity, so an action to open the new activity form is returned [2].

1: https://github.com/odoo/odoo/blob/14.0/addons/mail/models/mail_activity.py#L523-L536
2: https://github.com/odoo/odoo/blob/14.0/addons/mail/models/mail_activity.py#L484-L485

Reply:
@seb-odoo yes there are two cases for `action_feedback_schedule_next` 

1. The action_feedback creates its own next activity (for example when `force_next` is `True`[1]). In that case `action_feedback_schedule_next` will return `False` [2].
2. The action_feedback doesn't aumatically create a new activity, so an action to open the new activity form is returned [2].

The idea behind that seems to be that when the next activities are automatically generated, there is no need of an action to manually create the next one.

1: https://github.com/odoo/odoo/blob/14.0/addons/mail/models/mail_activity.py#L523-L536
2: https://github.com/odoo/odoo/blob/14.0/addons/mail/models/mail_activity.py#L484-L494